tests/software/muscle/main.nf

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env nextflow
+
+nextflow.enable.dsl = 2
+
+include { MUSCLE } from '../../../software/muscle/main.nf' addParams( options: ['args': '-fasta -verbose'])
+include { MUSCLE as MUSCLE_TREE } from '../../../software/muscle/main.nf' addParams( options: ['args': '-maketree'])
+
+workflow test_muscle {
+
+    input = file('https://raw.githubusercontent.com/nf-core/test-datasets/bcellmagic/testdata-bcr/C_primers.fasta', checkIfExists: true)
```

Comment:
It would be nice to have a file on the modules test data repository for testing, of course.
You just need a mult-fasta file, right? I just had a look and we have a multi fasta file here:
https://github.com/nf-core/test-datasets/blob/modules/data/genomics/sarscov2/illumina/fasta/contigs.fasta

It's contigs so not really suitable for an MSA of course, but as long as MUSCLE doesn't crash it should be fine.
Could you have a look at that? Otherwise we could add test data for this. Should be fairly straight forward.

Reply:
Hi @KevinMenden! 

Thanks a lot for taking a look. I first tried the fasta files available on the modules test data repo, but unfortunately MUSCLE was crashing so I selected something from another test-data repo. It's no problem at all for me to create this test data; I'll let you know when it is ready

Thanks again for the help!

Reply:
Amazing, thanks!

Reply:
Hi @KevinMenden 

Had another look at the modules test repo and found something suitable (I somehow missed `informative_sites_fas` first time round!!!). Hope this is ok and does the trick.